ebuild-writing/functions/src_test/text.xml

```diff
@@ -82,7 +82,6 @@ portage environment. Reasons for such a failure could include:
 </p>
 
 <ul>
-  <li>Needing to use X.</li>
```

Comment:
Is this related to the comment `patched out the tests that don't work without X.` in PR 4544? Instead of removing this sentence, should we expand it with this?

```
Reasons for such a failure could include:
- Needing to use X (Unless the test system can be configured/patched to exclude those certain tests).
```